docs/release/release_0_4_16.md

```diff
@@ -26,15 +26,38 @@ After discussion in [#4102](https://github.com/napari/napari/pull/4102) and [#41
 
 
 - Calling `viewer.open` *without* passing a plugin will result in an error if you have not saved a reader preference for that file pattern *and* multiple plugins can claim the file
-    - You can address this error by associating a preference for the file pattern, or calling `viewer.open(file_path, plugin=...)
-- A preferred reader failing to read your file will result in an error
+    - To save a preference for a file pattern in Python, use:
+
+    ```python
+    from napari.settings import get_settings
+    get_settings().plugins.extension2reader['*.tif'] = napari_tifffile
+    get_settings().plugins.extension2reader['*.zarr'] = napari_ome_zarr
+    ```
+
+    - To specify a plugin in a Python script:
+
+    ```python
+    import napari
+
+    viewer = napari.Viewer()
+    viewer.open('my-path.tif') # this will throw MultipleReaderError if napari_tifffile is installed as both it and builtins could open the file
+    viewer.open('my-path.tif', plugin=napari_tifffile) # this won't
```

Comment:
same here... shouldn't the plugin be a string?

Reply:
> # this will throw MultipleReaderError if napari_tifffile is installed as both it and builtins could open the file

apologies if I'm forgetting the conversation we had about this, and I know this is from a previous PR... but we decided against letting builtins take it programatically?  Seems a bummer that installing a plugin that recognizes tiff or png or something now means that a simple programatic `viewer.open('something.tif')` is now a full on error

Reply:
Well, we didn't decide anything yet @tlambert03. I remember at the hackathon we said maybe `builtins` should take it, but @jni's opinion in #4389 (and I tend to agree), is that if we do this at all, it should be the *other* plugin that takes the file.

Do you think deciding on this is critical before release? Fwiw the change itself would be trivial, it's just a matter of whether we want to make it. The fact that your suggestion is exactly opposite to Juan's makes me think that maybe this is a bit of magic we should just steer away from? I totally get the reticence and the feeling of "I just installed the plugin obviously I want to use it", but I just don't know if that's enough reason to sprinkle a bit of the ol' magic back in.